app/assets/javascripts/discourse/models/composer.js.es6

```diff
@@ -762,16 +768,21 @@ const Composer = RestModel.extend({
 
     const props = {
       raw: this.get("reply"),
+      raw_old: !post.get("wiki") || this.get("editConflict") ? null : this.get("originalText"),
```

Comment:
Any reasons we're not **always** sending the `originalText`? Conflicts are not limited to wiki posts.

```suggestion
      raw_old: this.get("originalText"),
```

Reply:
The absence of `raw_old` will skip the check on server side.

First time `raw_old` will be sent so the server can check if there is a problem. The second time (after a `editConflict` happened) `raw_old` will not be sent anymore which will bypass the check and overwrite the content.

However, I do agree that it could work for all posts, not just wiki. The spec mentioned only these and I meant to ask you about it. :)